src/securejoin.rs

```diff
@@ -1306,4 +1325,111 @@ mod tests {
             "vc-contact-confirm-received"
         );
     }
+
+    #[async_std::test]
+    async fn test_secure_join() {
+        let alice = TestContext::new_alice().await;
+        let bob = TestContext::new_bob().await;
+
+        let chatid = chat::create_group_chat(&alice.ctx, VerifiedStatus::Verified, "the chat")
+            .await
+            .unwrap();
+
+        // Generate QR-code, secure-join implied by chatid
```

Comment:
There is `securejoin` in the function name too. Does this comment mean that verified group instead of verified contact is used, because chat id points to the group chat?

Reply:
yeah, the naming here is a bit funny and the API certainly still C-style.  The entire module is also called "securejoin".  I might eventually rename this, maybe something like "join_invite" or so.  But in one the the followup PRs